pkgs/build-support/php/build-composer-project.nix

```diff
@@ -29,38 +29,9 @@ let
       patches = previousAttrs.patches or [ ];
       strictDeps = previousAttrs.strictDeps or true;
 
-      # Should we keep these empty phases?
```

Comment:
IIRC, the point is to prevent running `./configure`, `make` & co., while still being able to `overrideAttrs` like in `mkDerivation`-based packages (e.g. without having to also override `dontConfigure` back to `false`). It should probably be documented.

Reply:
@SuperSandro2000 Do you plan to do something in here?

Reply:
Do the packages even contain files which would be run?

Reply:
What do you mean?